README.md

```diff
@@ -52,4 +56,10 @@
 ## C++
 
   * [Doxygen](http://www.stack.nl/~dimitri/doxygen)
- 
\ No newline at end of file
+  
+## Python
+  * [Pydoc](http://pydoc.org/)
+  * [Pydoctor](https://launchpad.net/pydoctor)
+  * [Doxygen - can create documentation in various formats (HTML, LaTeX, PDF, ...)](http://www.doxygen.org/)
```

Comment:
I know it handles other languages, but I am categorizing them according to the language setup. Can you use python to install Doxygen on your machine?